packages/eslint-plugin/src/rules/no-floating-promises.ts

```diff
@@ -95,6 +100,20 @@ export default util.createRule<Options, MessageId>({
             context.report({
               node,
               messageId: 'floating',
+              suggest: [
+                {
+                  messageId: 'floatingFixAwait',
+                  fix(fixer): TSESLint.RuleFix {
+                    let code = sourceCode.getText(node);
+                    code = `await ${
+                      code.startsWith('void')
```

Comment:
I don't see anything in the tests about the code starting with `'void'`. Could you add a few tests please?

At the very least:
* See that `void returnsPromise();` gets turned into `await returnsPromise();`
    * A variation or two of this that uses `/* ... */` comments or similar after the `void` - to make sure text ranges are accounted for well
* See that something coincidentally starting with `void`, like `voidReturnsPromise();`, is understood to not actually be using the `void` operator
    * Tip: you'll probably want to check the AST, such as seeing whether this is in a [UnaryExpression whose operator is `"void"`](https://typescript-eslint.io/play/#ts=4.8.4&jsx=true&sourceType=module&showAST=es&code=G4ewlgJgBA7ghgFwBQEoDcQ&eslintrc=N4KABGBEBOCuA2BTAzpAXGYBfEWg&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA)

Reply:
Of course.
I added three tests that check `void returnsPromise();` gets turned into `await returnsPromise()`; .

Are your requirements being met?